app/templates/gulpfile.js

```diff
@@ -112,6 +122,22 @@ gulp.task('serve:dist', function () {
   });
 });
 
+gulp.task('serve:tests', function(){
+  browserSync({
+    notify: false,
+    port: 9090,
+    server: {
+      baseDir: 'test'
+    }
+  });
```

Comment:
I omitted the route for the local `bower_components` directory as `generator-mocha` uses relative URLs inside `test/index.html`. Haven't checked with Jasmine yet...